php/blocks/block-coauthor-avatar/class-block-coauthor-avatar.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Co-Author Avatar
+ *
+ * @package Automattic\CoAuthorsPlus
+ * @since 3.6.0
+ */
+
+namespace CoAuthors\Blocks; 
+
+use WP_Block;
+/**
+ * Block CoAuthor Avatar
+ *
+ * @package CoAuthors
+ */
+class Block_CoAuthor_Avatar {
+
+	/**
+	 * Register Block
+	 *
+	 * @since 3.6.0
+	 */
+	public static function register_block(): void {
+		register_block_type(
+			dirname( COAUTHORS_PLUS_FILE ) . '/build/blocks/block-coauthor-avatar',
+			array(
+				'render_callback' => array( __CLASS__, 'render_block' ),
+			)
+		);
+	}
+	/**
+	 * Render Block
+	 *
+	 * @since 3.6.0
+	 * @param array    $attributes
+	 * @param string   $content
+	 * @param WP_Block $block
+	 * @return string
+	 */
+	public static function render_block( array $attributes, string $content, WP_Block $block ): string {
+
+		$author = $block->context['co-authors-plus/author'] ?? array();
+
+		if ( empty( $author ) ) {
+			return '';
+		}
+
+		$avatar_urls = $author['avatar_urls'] ?? array();
```

Comment:
@GaryJones Question here related to PHP version compatibility standards - would we want to remove usage of the `??` null coalescing operator [added in PHP 7](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)? [The README](https://github.com/Automattic/Co-Authors-Plus/blob/develop/README.md) still shows:

> Requires PHP: 5.6

I'm not sure if it makes more sense to try to keep that 5.6 compatibility, or to bump the minimum supported version up. What do you think makes sense here?

Reply:
Combining these questions together. From the `render_block()` definition a few lines up:

https://github.com/Automattic/Co-Authors-Plus/blob/5638d641eb7ae8ed469a25c3c857f7a427e40fcf/php/blocks/block-coauthor-avatar/class-block-coauthor-avatar.php#L41

Same PHP compatibility question for the scalar and return value type hinting. Should these be modified to be PHP 5.6 compatible?

Reply:
One more, also seeing nullable types a bit later:

https://github.com/Automattic/Co-Authors-Plus/blob/5638d641eb7ae8ed469a25c3c857f7a427e40fcf/php/blocks/templating/class-templating.php#L23

These appear to be a [PHP 7.1 feature](https://www.php.net/manual/en/migration71.new-features.php).

Reply:
No - #954 will be bumping the min PHP to 7.4 in the same minor release as this PR will go into.